README.org

```diff
@@ -636,7 +636,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
 
 *** Java
 
-    - [[https://github.com/emacs-eclim/emacs-eclim][emacs-eclim]] - An Eclipse plugin which exposes Eclipse features through a server interface.
+    - [[https://github.com/emacs-eclim/emacs-eclim][emacs-eclim]] - An Eclipse plugin that exposes Eclipse features through a server interface.
```

Comment:
The original repo (which points towards lsp-java now...) uses the same previous grammar. 

I'm fine with keeping the new changes or keeping the old wording. Either or. Thanks for the catch.